episodes/chapter5.md

```diff
@@ -0,0 +1,302 @@
+]
+```
+
+Getting the status of an organization node for a given collaboration: The following returns the nodes (there is a node for each collaboration the organization contributes to).
+
+``` python
+>>> client.node.list(organization=4)
+
+[
+    {'config': [], 'last_seen': '2024-06-18T06:57:48.114072', 
+    'organization': {'id': 4, 'link': '/api/organization/4', 
+    'methods': ['PATCH', 'GET']}, 
+    'collaboration': {'id': 4, 'link': '/api/collaboration/4', 
+    'methods': ['GET', 'PATCH', 'DELETE']}, 
+    'name': 'PhY-research - LIFELINES_organization', 'status': 'offline', 
+    'type': 'node', 'ip': None, 'id': 13}, 
+    {'config': [], 'last_seen': None, 
+    'organization': {'id': 4, 'link': '/api/organization/4', 
+    'methods': ['PATCH', 'GET']}, 
+    'collaboration': {'id': 5, 'link': '/api/collaboration/5', 
+    'methods': ['GET', 'PATCH', 'DELETE']}, 
+    'name': 'GHT_consortium - LIFELINES_organization', 'status': None, 
+    'type': 'node', 'ip': None, 'id': 18}
+]
+```
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: challenge
+## Challenge 3: Creating a task that runs a partial algorithm**
+
+The following is the information of a new algorithm **(TODO a new algorithm should be created for this)**, that you need to use on the collaboration whose 100% of its nodes are active:
+
+- Image: harbor2.vantage6.ai/**TODO**alg_xxyy
+- Functions: central(), partial()
+- Algorithm source code: github.com/**TODO**/repo/alg_xxyy
+
+Using the guidelines given in the __Python client - basic concepts__ section above, use the client to create a task for executing the `partial()` function on all the nodes. Check in the source code what the parameters of this function are, and define their input accordingly. For the database selection, all the nodes use the one labeled as 'default'.
+
+Include the necessary code to wait until the new task is completed, and to get and print the results. Discuss the output with the instructors.
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: challenge
+## Challenge 4: Create a task that runs a central  function, and figure out why it failed**
+
+Using the Python client, create a new task with the same algorithm from Challenge #3. This time, launch the `central()` function. As you will see, this time the task failed. Using the client, try to find out which of the nodes involved on the task execution failed and the error message. Discuss with the instructors, based on the algorithm source code and the error message the cause of the error, and what should be done to fix it.
+**(TODO This requires the new algorithm or the data on one of the nodes to cause an error)**
+**TODO: I'm still not sure how to do this with the v6 client - this would be included at the beginning**
```

Comment:
This should be done by looking into status field of the run/result and printing their log files. If you don't want to do this before merging, could you create an issue for this?

Reply:
Issue #36 created.